app/views/install/compose.phtml

```diff
@@ -595,7 +595,7 @@ services:
       - _APP_REDIS_PASS
 
   mariadb:
-    image: mariadb:10.8.3 # fix issues when upgrading using: mysql_upgrade -u root -p
+    image: mariadb:10.7 # fix issues when upgrading using: mysql_upgrade -u root -p
```

Comment:
Is this intended ?

Reply:
The new version broke my setup, same for Damodar. Not sure what to do with it.

Reply:
Okay 👍